Cargo.toml

```diff
@@ -17,6 +17,7 @@ svgtypes = "0.8.0"
 tiny-skia = "0.6.1"
 fontdb = "0.7.0"
 log = "0.4.11"
+env_logger = "0.8.4"
```

Comment:
Can we use 0.9.0? https://github.com/env-logger-rs/env_logger/releases/tag/v0.9.0